client/cmake/android.cmake

```diff
@@ -1,6 +1,6 @@
 message("Client android ${CMAKE_ANDROID_ARCH_ABI} build")
 
-set(APP_ANDROID_MIN_SDK 24)
+set(APP_ANDROID_MIN_SDK 26)
```

Comment:
Please don't drop Android 7 support without any valuable reason for it.
It's still around 3% of all devices (source: https://www.appbrain.com/stats/top-android-sdk-versions).

Reply:
We're not going to drop Android 7. The version for 7 will be in another branch.

Reply:
Great news!
Thanks a lot for the response!